api/swagger.yaml

```diff
@@ -6468,6 +6555,31 @@ paths:
             format: "binary"
       tags:
         - "Plugins"
+  /plugins/{name}/push:
+    post:
+      summary: "Push a plugin"
+      operationId: "PluginsPush"
```

Comment:
Could we make this `PluginPush` ? I've been normalizing everything to be singular.